pkg/engine/internal/imageverifier.go

```diff
@@ -280,6 +283,16 @@ func (iv *ImageVerifier) verifyImage(
 		return engineapi.RuleError(iv.rule.Name, engineapi.ImageVerify, fmt.Sprintf("failed to add image to context %s", image), err, iv.rule.ReportProperties), ""
 	}
 	if len(imageVerify.Attestors) > 0 {
+		for _, exception := range matchedExceptions {
+			if exception.Spec.VerifyImages != nil {
+				for _, imageRef := range exception.Spec.VerifyImages {
+					if matchReferences(imageRef.ImageReferences, image) {
+						iv.logger.Info("skipping image verification", "image", image, "policy", iv.policyContext.Policy().GetName(), "ruleName", iv.rule.Name, "exception", exception.Name)
+						return engineapi.RuleSkip(iv.rule.Name, engineapi.ImageVerify, fmt.Sprintf("skipping image reference image %s, policy %s ruleName %s, exception %s", image, iv.policyContext.Policy().GetName(), iv.rule.Name, exception.Name), iv.rule.ReportProperties), ""
+					}
+				}
+			}
+		}
```

Comment:
This logic should also work for attestations not just attestors, it should be outside the length check